tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -1881,18 +2487,18 @@ TEST_F( HVACFixture, VRFTest_SysCurve_WaterCooled ) {
 		DataSizing::CurZoneEqNum = 1;
 		DataEnvironment::OutBaroPress = 101325; // sea level
 		DataZoneEquipment::ZoneEquipInputsFilled = true; // denotes zone equipment has been read in
-		StdRhoAir = PsyRhoAirFnPbTdbW( DataEnvironment::OutBaroPress, 20.0, 0.0 );
-		ZoneEqSizing.allocate( 1 );
-		ZoneSizingRunDone = true;
-		ZoneEqSizing( CurZoneEqNum ).DesignSizeFromParent = false;
-		ZoneEqSizing( CurZoneEqNum ).SizingMethod.allocate( 25 );
-		ZoneEqSizing( CurZoneEqNum ).SizingMethod( DataHVACGlobals::SystemAirflowSizing ) = DataSizing::SupplyAirFlowRate;
+		DataEnvironment::StdRhoAir = PsyRhoAirFnPbTdbW( DataEnvironment::OutBaroPress, 20.0, 0.0 );
+		DataSizing::ZoneEqSizing.allocate( 1 );
+		DataSizing::ZoneSizingRunDone = true;
+		DataSizing::ZoneEqSizing( CurZoneEqNum ).DesignSizeFromParent = false;
+		DataSizing::ZoneEqSizing( CurZoneEqNum ).SizingMethod.allocate( 25 );
+		DataSizing::ZoneEqSizing( CurZoneEqNum ).SizingMethod( DataHVACGlobals::SystemAirflowSizing ) = DataSizing::SupplyAirFlowRate;
```

Comment:
I don't care either way but I thought it better highlighted which 
variable was used.

On 2015-09-09 16:14, Michael J. Witte wrote:
> In tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc [1]:
> 
>> @@ -1881,18 +2487,18 @@ TEST_F( HVACFixture,
> VRFTest_SysCurve_WaterCooled ) {
>> DataSizing::CurZoneEqNum = 1;
>> DataEnvironment::OutBaroPress = 101325; // sea level
>> DataZoneEquipment::ZoneEquipInputsFilled = true; // denotes zone
> equipment has been read in
>> - StdRhoAir = PsyRhoAirFnPbTdbW( DataEnvironment::OutBaroPress,
> 20.0, 0.0 );
>> - ZoneEqSizing.allocate( 1 );
>> - ZoneSizingRunDone = true;
>> - ZoneEqSizing( CurZoneEqNum ).DesignSizeFromParent = false;
>> - ZoneEqSizing( CurZoneEqNum ).SizingMethod.allocate( 25 );
>> - ZoneEqSizing( CurZoneEqNum ).SizingMethod(
> DataHVACGlobals::SystemAirflowSizing ) =
> DataSizing::SupplyAirFlowRate;
>> + DataEnvironment::StdRhoAir = PsyRhoAirFnPbTdbW(
> DataEnvironment::OutBaroPress, 20.0, 0.0 );
>> + DataSizing::ZoneEqSizing.allocate( 1 );
>> + DataSizing::ZoneSizingRunDone = true;
>> + DataSizing::ZoneEqSizing( CurZoneEqNum ).DesignSizeFromParent =
> false;
>> + DataSizing::ZoneEqSizing( CurZoneEqNum ).SizingMethod.allocate( 25
> );
>> + DataSizing::ZoneEqSizing( CurZoneEqNum ).SizingMethod(
> DataHVACGlobals::SystemAirflowSizing ) =
> DataSizing::SupplyAirFlowRate;
> 
> @rraustad [2] Are you intentionally adding the namespaces to these?
> They aren't used that way in the other unit test in this file. I'm in
> the midst of cleaning up so stray indentation changes in the main
> routine and will clean these up also if you agree.
> 
> --
> Reply to this email directly or view it on GitHub [3].
> 
> 	*
> 
> Links:
> ------
> [1] https://github.com/NREL/EnergyPlus/pull/5164#discussion_r39090211
> [2] https://github.com/rraustad
> [3] https://github.com/NREL/EnergyPlus/pull/5164/files#r39090211